src/web/stash/ItemTooltip.tsx

```diff
@@ -14,18 +16,52 @@ export function ItemTooltip({ item }: { item: Item }) {
     return <span class={className}>{item.name}</span>;
   }
 
+  const rangeMap = getRanges(item).reduce(function (
+    map: { [prop: string]: number[] },
+    range
+  ) {
+    if (
+      range.min &&
+      range.max &&
```

Comment:
Correct me if I'm wrong, but I believe the latest version can't have min or max undefined on a `ModifierRange`. I think I forgot to update the type, so that's what I'd do instead of this extra check: remove the `?` from min and max in ModifierRange. If Typescript compiles and doesn't complain, it's pretty safe.